### PR TITLE
Pull Request from Capston-1-desktopAndMobile branch to Master

### PR DESCRIPTION
**Pull request by Nahid1911**
**Project Requirements are as follows**

- Should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

1.  Colors.
2. Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
3.  Layout: composition and space between elements.

- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You should build only these 2 pages: (The home and the about page)
- Each of these pages should have versions for 2 different screen sizes: (Mobile max 768px and for others min 768px).
- Interactions:

1. Links
     - The home page should have a link in the menu to the about page.
     - The logo in the header links to the home page.
  2. Mobile menu:
     - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
     - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
     - The mobile menu should have a close (X) button that closes the menu.
    
  3. Dynamic page:
     - The section "Featured speakers" should be created dynamically in JavaScript.
     - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
  
  4. Video description of the [Project](https://www.loom.com/share/d5d5c4548bc5479aa11aa401e4d87466).

**for more information please visit the [requirements page](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,293 +1,94 @@
...
-        <div class="btn-container">
-          <button id="4" type="submit" class="btn-all">See project</button>
-        </div>
-      </div>
-    </div>
-    <div class="project">
-      <div class="project-preview">
-        <img src=${workHistory[0].cardThumbnail}>
+const speaker = document.getElementById('speakersContainer');
+speaker.innerHTML = `
```
Comment (mukhammadrizooff) [inline script.js:17]: - [ ] [REQUIRED] Good job so far! Please give the style to the `Speakers` section as shown in [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). There should be a `chess background shape` behind the avatar picture of each speaker and the `background color` of the whole section should be white. Please correct them based on what I said above 🙏🏼 

### Desktop
image

### Mobile
image

File: index.html [hunk 2]
```diff
@@ -4,173 +4,154 @@
...
-      <section id="WorkContainerID"class="work-container">
-        <div class="my-work-text">
-          <div id="portfolio" class="my-work-text-heading">
-            <h2>My Recent Works</h2>
-          </div>
-          <div class="hr-line">
-            <hr>
+    </main>
+    <section class="mainProgram flex">
+      <p>Main Program</p>
```
Comment (mukhammadrizooff) [inline index.html:79]: - [ ] [REQUIRED] Good job so far! In both `desktop` and `mobile` designs, give the correct `color` given in Behance design to the `heading` text and `icons` in the program cards. And pay attention to the `distance` between the heading texts in `Mobile design`, some of them are stuck together 🙏🏼 

### Desktop

image

### Mobile

image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    <footer class="footerDiv flex">
+      <div class="footerlogo flex">
+        <img src="assets/logo_Porabo_live.jpg" alt="Logo of the company">
+      </div>
+      <div class="addressAndCredit">
+        <address> 2015 Creative Commons Korea, Some Rights Reserved</address>
+        <p>Developed by <em>Nahid Raihan</em></p>
+        <p>Template credit <a class="no-decoration" href="https://www.behance.net/adagio07"><em>Cindy Shin</em></a></p>
+      </div>
+    </footer>
```
Comment (mukhammadrizooff) [inline about.html]: - [ ] [REQUIRED] Good job so far! Please pay attention to the differences in the images I've highlighted below and correct them`!` Change the `background color` of this `section` and `footer` to the correct color and give the required `filter` and `style` to the `images` in the section and `adjust` the position of the `elements` 🙏🏼 

### Desktop
image

### Mobile
image

File: README.md [hunk 4]
```diff
@@ -31,46 +31,47 @@
...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-To see the live demo of the page please click <a href="https://nahid1911.github.io/Nahid-s_Portfolio/">here</a>:
+<!-- To see the live demo of the page please click <a href="https://nahid1911.github.io/Nahid-s_Portfolio/">here</a>: -->
+The Page is under consturctin
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] The project deployed properly as expected but a `link` to the deployed project isn't present in the `README` please paste it 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Nahid1911, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 
- Descriptive commit history ✔️ 
- Descriptive `PR` description ✔️ 


## Required Changes ♻️
- [ ] [REQUIRED] You should record a `video` presenting the features of the project. The content of the video should include a description of the project 🙏🏼 

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Nahid1911) [reply script.js:17]: @mukhammadrizooff 
the issue resolved

File: index.html (hunk 2 above)
Comment (Nahid1911) [reply index.html:79]: @mukhammadrizooff 
Resolved

File: about.html (hunk 3 above)
Comment (Nahid1911) [reply about.html]: @mukhammadrizooff 
Resolved

File: README.md (hunk 4 above)
Comment (Nahid1911) [reply README.md]: @mukhammadrizooff 
resolved

Comment (Nahid1911) [reply README.md]: @mukhammadrizooff 
Resolved

**Comment by Nahid1911** [comment]
Video description of the project has been added in the PR description.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Nahid1911 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**